### ISSUE TITLE
Search and Navigate directly to a skill input does not have an associated label or aria-label attribute

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1207]
Input field needs an associate label or aria-label attribute to indicate what the field's intent is to screenreader users.

**Comment by rmmayo** [comment]
completed as part of #190
